### ISSUE TITLE
uassert/urequire: Messages from PanicsWithMessage method are inverted

### ISSUE THREAD

**Issue by ajnavarro** [gnolang/gno#2503]
### Description

When a panic with a specific message is expected but the output is not the same, the test error message is wrong.

### Steps to reproduce

Use the PanicsWithMessage method from uassert/urequire.

### Expected behavior

Executing:

```
urequire.PanicsWithMessage(t, "ppp", func() {
		ExecuteProposal(pid)
	})

```

Output should be:

```
=== RUN   TestPackage
func should panic with message: ppp
        Panic value:   OTHER PANIC VALUE
```

### Actual behavior

Executing:

```
urequire.PanicsWithMessage(t, "ppp", func() {
		ExecuteProposal(pid)
	})

```

Output is:

```
=== RUN   TestPackage
func should panic with message: OTHER PANIC VALUE
        Panic value:   ppp
```

**Comment by linhpn99** [comment]
Here is hotfix PR https://github.com/gnolang/gno/pull/2506

**Comment by zivkovicmilos** [comment]
Closed by #2506 

Thank you @linhpn99 for the fix 🙏
